### PR TITLE
Part 1 - Started updating client to tap

### PR DESCRIPTION
**Pull request by lancetipton**
This is part 1 of a 4 part update. I tried to break it up into working chunks, but its difficult because of the re-name, and some changes are required for other parts to work.
I recommend you go through each part and check the code. Once you've gone through it all, ping me and we can go over a few things.

**Updates**
* Started convert `client` to `tap`
* Updated `app.json` client refs to `tap`

### CODE REVIEW

File: src/buildConstants.js [hunk 1]
```diff
@@ -115,40 +115,40 @@ const buildDynamicContent = (appConfig={}) => {
...
   
   // Ensure the required app data exists
   validateApp(appRoot, appConfig)
 
-  // Setup the client, to get the correct path data
+  // Setup the tap, to get the correct path data
   const {
     APP_CONFIG_PATH,
     BASE_PATH,
     CLIENT_NAME,
```
Comment (mpcarolin) [inline src/buildConstants.js]: shouldn't these constants be updated as well? CLIENT_NAME to TAP_NAME, for instance?

File: src/setup.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ module.exports = (appRoot, appConfig, contentResolver, clientName) => {
     DYNAMIC_CONTENT,
     EXTENSIONS,
     HAS_CLIENT
```
Comment (mpcarolin) [inline src/setup.js]: same as the previous comment, wouldn't HAS_CLIENT and other constants change?

File: src/setupTap.js [hunk 3]
```diff
@@ -6,56 +6,56 @@ const { validateApp } = require('./helpers')
...
+ * Get the name of the active tap from the passed in param, ENV, app.json config
+ * @param {string} tapName - name of the active tap
  * @param {Object} appConfig - mobile keg app.json config
  *
- * @returns {string} - name of the active client
+ * @returns {string} - name of the active tap
  */
-const getActiveClientName = (appConfig, clientName) => {
+const getActiveTapName = (appConfig, tapName) => {
   const { CLIENT } = process.env
```
Comment (mpcarolin) [inline src/setupTap.js]: wouldn't this also be: `const { TAP } = process.env`?

File: src/setupTap.js [hunk 4]
```diff
@@ -6,56 +6,56 @@ const { validateApp } = require('./helpers')
...
- * Gets the location of the active clients directory
- * <br> First checks root_dir/node_modules/zr-rn-clients/CLIENT_NAME
- * <br> Next checks root_dir/clients/CLIENT_NAME
+ * Gets the location of the active taps directory
+ * <br> First checks root_dir/node_modules/zr-rn-taps/CLIENT_NAME
+ * <br> Next checks root_dir/taps/CLIENT_NAME
  * @param {string} appRoot - Root directory of the mobile keg
  * @param {Object} appConfig - Root app.json config
- * @param {string} CLIENT_NAME - Name of the active client
+ * @param {string} CLIENT_NAME - Name of the active tap
```
Comment (mpcarolin) [inline src/setupTap.js]: `TAP_NAME`?

File: src/setupTap.js [hunk 5]
```diff
@@ -77,16 +77,16 @@ const cleanupOldTempConfig = TEMP_FOLDER_PATH => {
 }
 
 /**
- * Joins the app config root with the clients config
+ * Joins the app config root with the taps config
  * <br> Writes the joined config to disk inside a temp folder
  * @param {Object} appConfig - default app.json config
- * @param {string} CLIENT_PATH - Path to the clients folder
+ * @param {string} CLIENT_PATH - Path to the taps folder
```
Comment (mpcarolin) [inline src/setupTap.js]: `TAP_PATH`?

File: src/setupTap.js [hunk 6]
```diff
@@ -109,77 +109,77 @@ const buildJoinedConfigs = (appConfig, CLIENT_PATH, TEMP_FOLDER_PATH) => {
...
 /**
- * Looks up the clients app.json file and joins it with the default app.json
+ * Looks up the taps app.json file and joins it with the default app.json
  * <br> Writes the joined config to disk inside a temp folder
  * @param {string} appRoot - Root directory of the mobile keg
  * @param {Object} appConfig - default app.json config
- * @param {string} CLIENT_PATH - path to the client folder
- * @param {boolean} HAS_CLIENT - if an active client is set
+ * @param {string} CLIENT_PATH - path to the tap folder
+ * @param {boolean} HAS_CLIENT - if an active tap is set
```
Comment (mpcarolin) [inline src/setupTap.js]: here too

File: src/setupTap.js [hunk 7]
```diff
@@ -109,77 +109,77 @@ const buildJoinedConfigs = (appConfig, CLIENT_PATH, TEMP_FOLDER_PATH) => {
...
+    ? getTapPath(appRoot, appConfig, CLIENT_NAME)
     : BASE_PATH
 
   // Get the path to the app config ( either the appConfig or joined temp config )
-  const { APP_CONFIG, APP_CONFIG_PATH } = setupClientConfig(
+  const { APP_CONFIG, APP_CONFIG_PATH } = setupTapConfig(
     appRoot,
     appConfig,
     CLIENT_PATH,
     HAS_CLIENT
```
Comment (mpcarolin) [inline src/setupTap.js]: and here

**Comment by lancetipton** [comment]
Closing, merge through this commit => https://github.com/simpleviewinc/sv-tap-resolver/commit/d60c0ee4af3c0f5d330d74774d1eadbd489cf33d

**Pull request closed without merging**